<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="mapLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="toolsLayout">
        <item>
         <widget class="QLabel" name="toolsLabel">
          <property name="text">
           <string>Tools</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editTilesButton">
          <property name="text">
           <string>Edit Tiles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editProblemButton">
          <property name="text">
           <string>Edit Problem</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="toolsSeparator">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="algorithmsLabel">
          <property name="text">
           <string>Algorithms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="aStarButton">
          <property name="text">
           <string>A*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="thetaStarButton">
          <property name="text">
           <string>Theta*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hpaStarButton">
          <property name="text">
           <string>HPA*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hptStarButton">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="algorithmsSeparator">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="toolSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="mapScaleSlider">
        <property name="value">
         <number>75</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mapViewLayout">
        <item>
         <widget class="QLabel" name="mapLabel">
          <property name="text">
           <string>Map</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="mapView">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="scenarioLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="scenarioLabel">
        <property name="text">
         <string>Scenario</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="scenarioList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="scenarioButtonLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="scenarioEditButtonLayout">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>Add...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="scenarioRunButtonLayout">
          <item>
           <widget class="QPushButton" name="runSelectedButton">
            <property name="text">
             <string>Run Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runAllButton">
            <property name="text">
             <string>Run All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Map"/>
    <addaction name="actionLoad_Multiple_Mapds"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Scenario"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Map">
   <property name="text">
    <string>Load Map..</string>
   </property>
  </action>
  <action name="actionLoad_Multiple_Mapds">
   <property name="text">
    <string>Load Multiple Maps...</string>
   </property>
  </action>
  <action name="actionLoad_Scenario">
   <property name="text">
    <string>Load Scenario..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mapScaleSlider</tabstop>
  <tabstop>mapView</tabstop>
  <tabstop>scenarioList</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>runSelectedButton</tabstop>
  <tabstop>runAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
